dimod/generators/wireless.py

```diff
@@ -0,0 +1,985 @@
+# Copyright 2023 D-Wave Systems Inc.
```

Comment:
I think I liked this file being named `mimo` more. But `wireless` is also fine :shrug: